app/pb_kits/playbook/pb_icon_value/_icon_value.jsx

```diff
@@ -1,21 +1,59 @@
-import React from 'react';
-import PropTypes from "prop-types";
+/* @flow */
 
-const propTypes = {
-  className: PropTypes.string,
-  id: PropTypes.string
-};
+import React from 'react'
+import {
+  Icon,
+  Body,
+} from '../'
 
-class IconValue extends React.Component {
-  render() {
-    return (
-      <div className="pb_icon_value">
-        <span>ICON VALUE CONTENT</span>
-      </div>
-    )
-  }
+import {
+  buildAriaProps,
+  buildDataProps,
+  buildCss,
+} from '../utilities/props'
+
+type IconValueProps = {
+  align: 'left' | 'center' | 'right',
+  aria: object,
+  className: string,
+  data: object,
+  icon: string,
+  id: number,
+  text: string,
 }
 
-IconValue.propTypes = propTypes;
+const IconValue = ({
+  align = 'left',
+  aria = {},
+  className,
+  data = {},
+  icon,
+  id,
+  text,
+}: IconValueProps) => {
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+  const css = buildCss({
+    'pb_icon_value_kit': true,
+    [align]: true,
+  })
```

Comment:
What do you think of updating `buildCss` implementation to the below? 

```js
export const buildCss = (...rules) => classnames(rules).replace(/\s/g, '_')
```

I did that on my PR because we don't actually use that `delimiter` param anywhere, which I think is good because we keep the standard for the CSS classes, so that param may be unnecessary after all.

If you did that you could do just 
```js
buildCss('pb_icon_value_kit', align)
```

Reply:
Sounds like a nice improvement 👍 

I'll give that a shot in the morning, thanks @rafbgarcia 👊

Reply:
Actually... The current implementation should already allow for:

```jsx
buildCss('pb_icon_value_kit', align)
```

Am I missing something?

Reply:
Yes, the way the function is currently implemented, `align` would be passed as `delimiter` and not joined with `'pb_icon_value_kit'`.